packages/switch/package.json

```diff
@@ -45,7 +45,15 @@
         "@spectrum-css/toggle": "^2.1.0"
     },
     "dependencies": {
+        "@open-wc/lit-helpers": "^0.3.10",
+        "@react-aria/switch": "^3.0.0-rc.2",
+        "@react-aria/toggle": "^3.0.0-rc.2",
+        "@react-stately/toggle": "^3.0.0-rc.2",
+        "@react-types/checkbox": "^3.0.0-rc.2",
         "@spectrum-web-components/checkbox": "^0.3.1",
+        "es-dev-commonjs-transformer": "^0.1.1",
+        "haunted": "^4.7.0",
+        "react": "npm:haunted",
```

Comment:
Is there a way to get this without the cross dependency? It block up less nuanced services like Stackblitz: 
![image](https://user-images.githubusercontent.com/1156657/83811014-07cea380-a687-11ea-9dc2-e346fc9b9a78.png) which wouldn't be the biggest loss, but is certainly something support I'd love for us to be able to keep if possible.

Reply:
Yeah not sure, I guess we can probably do a module resolve as part of a build process, but definitely a problem for buildless environments.